src/test/end-to-end/tests/system-admin/create-system-admin-test.ts

```diff
@@ -1,70 +1,59 @@
+        I.see(TEST_FIRST_NAME);
+        I.see(TEST_SURNAME);
+        I.see(email);
 
-Scenario('I as a system admin should not be able to create duplicated system admin account', async ({ I }) => {
-    const email = randomData.getRandomEmailAddress;
-    await createSystemAdminAccount(TEST_FIRST_NAME, TEST_SURNAME, email);
+        I.click('Change');
+        I.waitForText('Create system admin account');
+        I.fillField('#firstName', TEST_FIRST_NAME + '1');
+        I.click('Continue');
+        I.waitForText('Check account details');
+        I.click('Confirm');
+        I.waitForText('Account has been created');
+        I.see(TEST_FIRST_NAME);
+        I.see(TEST_SURNAME);
+        I.see(email);
 
-    I.loginAsSystemAdmin();
-    I.see('System Admin Dashboard');
-    I.createNewSystemAdminAndContinue(TEST_FIRST_NAME, TEST_SURNAME, email);
-    I.waitForText('Check account details');
-    I.click('Confirm');
-    I.waitForText('Account has been rejected');
-    I.see(
-        'This user already has an account. If the user requires a system admin account, their previous account will need to be deleted first before one can be created.'
-    );
-    I.logout();
-});
+        I.click('Home');
+        I.waitForText('System Admin Dashboard');
+        I.createNewSystemAdminAndContinue(TEST_FIRST_NAME, TEST_SURNAME, email);
+        I.waitForText('Check account details');
+        I.click('Confirm');
+        I.waitForText('Account has been rejected');
+        I.see(
+            'This user already has an account. If the user requires a system admin account, their previous account will need to be deleted first before one can be created.'
+        );
 
-Scenario(
-    'I as a system admin should not be able to create a new system admin if the maximum number of accounts has been reached',
-    async ({ I }) => {
-        // Continue creating new system admins until we see the 'above max system admin' error
-        let response = await createSystemAdminAccount(TEST_FIRST_NAME, TEST_SURNAME, randomData.getRandomEmailAddress);
-        while (!response?.error && !response?.aboveMaxSystemAdmin) {
-            response = await createSystemAdminAccount(TEST_FIRST_NAME, TEST_SURNAME, randomData.getRandomEmailAddress);
-        }
+        I.createMaxSystemAdminAccounts(TEST_FIRST_NAME, TEST_SURNAME);
 
-        I.loginAsSystemAdmin();
-        I.see('System Admin Dashboard');
-        I.createNewSystemAdminAndContinue(TEST_FIRST_NAME, TEST_SURNAME, randomData.getRandomEmailAddress);
+        I.click('Home');
+        I.waitForText('System Admin Dashboard');
+        I.createNewSystemAdminAndContinue(TEST_FIRST_NAME, TEST_SURNAME, randomData.getRandomEmailAddress());
         I.waitForText('Check account details');
         I.click('Confirm');
         I.waitForText('Account has been rejected');
         I.see('The maximum number of System Admin accounts has been reached.');
+
+        I.clearAllAccountsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
```

Comment:
If we are not changing the test prefix, do we need to call clearAllAccountsByTestPrefix here?

Reply:
This has been removed :)